spec/tests_from_xctestrun_spec.rb

```diff
@@ -140,4 +140,31 @@
       ]
     )
   end
+
+  describe 'when the action is given an explicit test_prefix' do
+    it 'invokces XCTestList with the given test_prefix' do
+      allow(File).to receive(:exist?).and_call_original
+      allow(File).to receive(:exist?).with('path/to/fake.xctestrun').and_return(true)
+      allow(File).to receive(:read).with('path/to/fake.xctestrun').and_return(File.read('./spec/fixtures/format-version-2.xctestrun'))
+      expect(XCTestList)
```

Comment:
What this is doing is "pretending to be the `xctest_list` class that receives the message, "hey, what tests do you have for this prefix?". You have it returning tests that are prefixed with test and stating that you expect to be called and return these mocked tests.

The expectation is fine, but since we don't really care what is being returned, I don't think that you need to add the:
```ruby
.and_return(
..
)
```
part.

Reply:
Unfortunately, it looks like we have to return at least an empty array to make sure the rest of the code can function as expected - is that all right?

Reply:
Oh, true. Yes, that should be fine.